### ISSUE TITLE
Hover effect won't close

### ISSUE THREAD

**Issue by ghost** [web-tiki/responsive-grid-of-hexagons#24]
I have a problem with the hover of the tiles as seen on the picture.

What can I do about it?

![test](https://user-images.githubusercontent.com/32872348/31666047-29691ad4-b34b-11e7-8946-c688a234d540.png)

**Comment by noeldevelops** [comment]
any solution to this?? I'm having the same issue...

**Comment by poi33** [comment]
@noeldevelops in what web browser is this happening? Is there any zoom on the page?

**Comment by noeldevelops** [comment]
@poi33 I tested in Chrome, Safari, and Firefox - no zoom. 
<img width="1067" alt="screen shot 2018-05-24 at 8 26 02 am" src="https://user-images.githubusercontent.com/20971470/40491944-a3c63630-5f2c-11e8-9522-ef1fd28791ac.png">

**Comment by poi33** [comment]
Where are you getting the black squares from?
The default colour should be rgba(0, 128, 128, 0.8); or #008080. 
Did you change anything of the values in the css stylesheet? (hexagons.css) 
If you have your own css rules overriding some of stylesheet rules it would explain the gap. 
I could not recreate this in Chrome or Firefox.

**Comment by noeldevelops** [comment]
Thanks for trying to help! 

I changed the background color of `.hex h1, .hex p` in the stylesheet to make it black. I also deleted the first css rule: 
```* {margin: 0; padding: 0;}```

Adding it back with dev tools doesn't seem to fix the problem, but it's a good starting point to think about, so thanks for that reminder... 

I've also changed the html a bit to make it work in Wordpress, but nothing structural. 

I will continue to troubleshoot and if I can make a snippet that recreates the error I will send it over.
